cfgrammar/src/lib/yacc/parser.rs

```diff
@@ -305,20 +310,29 @@ impl YaccParser {
     }
 
     fn parse_rules(&mut self, mut i: usize) -> YaccResult<usize> {
+        let mut duplicate_spans = Vec::new();
```

Comment:
It would be easier to work with IMHO if this was an attribute on `YaccParser` (we'll have more of these in the future, so having them all as parameters will get messy).

Reply:
Changed the name to `duplicate_rule_spans` while doing that, and should be fixed in https://github.com/softdevteam/grmtools/pull/307/commits/d5ffadc4e2c426092604332bbcbb81a7a8e91adc